src/bindings/python/src/openvino/utils/postponed_constant.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from typing import Callable, List
+import openvino
+
+"""Postponed Constant is a way to materialize a big constant only when it is going to be serialized to IR and then immediately dispose."""
+
+
+# `maker` is a function that returns ov.Tensor that represents a target Constant
+def make_postponed_constant(element_type: openvino.Type, shape: openvino.Shape, maker: Callable[[], openvino.Tensor]) -> openvino.Op:
+    class PostponedConstant(openvino.Op):
+        class_type_info = openvino.DiscreteTypeInfo("PostponedConstant", "extension")
```

Comment:
We are making a new `PostponedConstant` class each time this function is called. @akuporos, is it good or bad from OV Python API perspective? Is it better to move the class definition outside of this function and supply it with `shape`, `element_type`, and `maker` as attributes? These attributes won't be serializable, but it doesn't matter because this Op is not intended for serialization and dissolves in `serialize`. But it is probably a better solution.

Reply:
Yes, I think it'd  be better to move the class definition outside of this function.